main.css

```diff
@@ -0,0 +1,490 @@
+    padding-top: 106px;
+}
+
+.subtxt {
+    margin: 13px 0 51px;
+    font-family: 'Montserrat', sans-serif !important;
+}
+
+.tile-section-cta {
+    margin-bottom: 125px;
+}
+
+.tile-left-button {
+    margin-right: 15px;
+}
+
+.tile-right-button {
+    color: #141516;
+    border: 1px solid #141516 !important;    
+    min-width: 165px;
+    margin-left: 15px;
+    background: #fff;
+}
+
+.tile-list-subtitle {
+    margin: 0;
+    padding-left: 13px;
+}
+
+.tile-list-txt {
+    max-width: 230px;
+    margin-bottom: 70px;
+    font-weight: 500 !important;
+}
+
+.tile-list-item{
+    min-width: 240px;
+}
+
+.tile-list-box {
+    padding: 0 100px 60px;
+}
+
+.tile-list {
+    flex-wrap: wrap;
+}
+
+.section-background {
+    background-color: #141516;
+}
+
+.centered-title {
+    margin: 61px 0 68px 0;
+}
+
+.secondary-section-box {
+    margin-bottom: 215px;
+}
+
+.secondary-section-content {
+    padding-top: 29px;
+}
+
+.secondary-section-txt {
+    max-width: 440px;
+    margin: 26px 0;
+}
+
+.section-content-link {
+    font-weight: 500 !important;
+    padding-right: 12px;
+}
+
+.side-content-box {
+    position: relative;
+    padding: 30px 0 0 50px;
+}
+
+.button-group {
+    background-color: #fff;
+    z-index: 1;
+    min-width: 260px;
+    min-height: 260px;
+    position: absolute;
+    left: 0;
+    top: 0;
+    border-radius: 27px;
+}
+
+.grouped-button {
+    background-color: #F5F5F5;
+    border: none;
+    border-radius: 12px;
+    padding: 16px 11px;
+    min-width: 190px;
+    word-spacing: 70px;
+}
+
+.subsection-box {
+    margin-bottom: 230px;
+}
+
+.subsection-content {
+    padding-top: 18px;
+}
+
+.secondary-title {
+    font-weight: 600;
+    font-size: 36px;
+    line-height: 44px;
+    font-family: 'Montserrat', sans-serif;
+    margin: 0;
+}
+
+.subsection-img-box {
+    position: relative;
+    padding: 20px 15px 0 0;
+}
+
+.icon-position-box {
+    width: 64px;
+    height: 52px;
+    background-color: #F5F5F5;
+    position: absolute;
+    border-radius: 12px;
+    top: 0;
+    right: 0;
+}
+
+.subsection-title {
+    margin: 0;
+}
+
+.subsection-txt {
+    margin-bottom: 55px;
+}
+
+.secondary-cta {
+    display: flex;
+    margin-bottom: 90px;
+}
+
+.twin-button-left {
+    border: 1px solid #141516 !important;    
+    min-width: 165px;
+    margin-right: 15px;
+}
+
+.twin-button-right {
+    background-color: #141516;
+    border: 1px solid #fff !important;
+    padding: 13px 61px 13px 61px !important;
+    margin-left: 15px;
+}
+
+.relative-img-box {
+    position: relative;
+    margin: 0 0 50px 20px;
+    margin-bottom: 120px;
+}
+
+.absolute-image {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    z-index: 1;
+}
+
+.portfolio-title {
+    margin: 133px 0 0;
+}
+
+.portfolio-txt {
+    margin: 14px 0 100px 0;
+}
+
+.portfolio{
+    padding-left: 0;
+}
+
+@media screen and (max-width: 1725px) {
+    .portfolio-top :nth-child(4){
+        display: none;
+    }
+    .portfolio-bot :nth-child(4){
```

Comment:
czy bot to jest jakiś skrót? nie używamy skrótów plus przed dwukropkiem nie robimy spacji w pseudoelemencie